Libraries/WebSocket/WebSocket.windows.js

```diff
@@ -1,27 +1,52 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
  *
```

Comment:
Check if #253 is now fixed.

Reply:
No - still work to do here to support all kinds of headers.

---
In reply to: [169755860](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169755860) [](ancestors = 169755860)